testdata/default/cheats/ArbitraryCheatcodes.t.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.8.18;
+
+import "ds-test/test.sol";
+import "cheats/Vm.sol";
+
+contract ArbitraryCheatcodesTest is DSTest {
+    Vm vm = Vm(HEVM_ADDRESS);
+
+    int128 constant min = -170141183460469231731687303715884105728;
+    int128 constant max = 170141183460469231731687303715884105727;
+
+    function test_int128() public {
+        int256 val = vm.arbitraryInt(16);
+        assertGe(val, min);
+        assertLe(val, max);
+    }
+
+    function testFail_int128() public {
+        int256 val = vm.arbitraryInt(16);
+        assertGt(val, max);
+    }
+
+    function test_address() public {
+        address fresh_address = vm.arbitraryAddress();
+        assert(fresh_address != address(this));
+        assert(fresh_address != address(vm));
+    }
+
+    function test_arbitraryUints(uint8 x) public {
+        vm.assume(0 < x);
+        vm.assume(x <= 32);
+        uint256 freshUint = vm.arbitraryUint(x);
+
+        assert(0 <= freshUint);
+        if (x == 32) {
+            assert(freshUint <= type(uint256).max);
+        } else {
+            assert(freshUint <= 2 ** (8 * x) - 1);
+        }
+    }
+
+    function test_arbitrarySymbolicWord() public {
+        uint256 freshUint192 = vm.arbitraryUint(192);
+
+        assert(0 <= freshUint192);
+        assert(freshUint192 <= type(uint192).max);
+    }
+}
+
+contract ArbitraryBytesTest is DSTest {
+    Vm vm = Vm(HEVM_ADDRESS);
+
+    bytes1 local_byte;
+    bytes local_bytes;
+
+    uint256 constant length_limit = 72;
```

Comment:
You can probably remove the `length_limit` constant, since Foundry does not need these restrictions. In Kontrol, we use this limit to ensure it will only execute the loop copying 32-by-32 bytes three times in the test.

Reply:
yep, removed, constant wasn't even used in tests